### ISSUE TITLE
v2.0.1: no timezones for RO country

### ISSUE THREAD

**Issue by mariusa** [manuelmhtr/countries-and-timezones#8]
Hola,

v2.0.1 returns null as timezones for RO country.
https://www.timeanddate.com/time/zone/romania

**Comment by mariusa** [comment]
I've found the bug. Because there's no single API call to get both country name and countries, I was doing
```
    country = ct.getCountry('RO')
    country.timezones = ct.getTimezonesForCountry('RO')
```
Looks like ct.getCountry() returns object by reference, which means that it breaks when one overwrites the .timezones property.
You might want to return a copy instead? If not, that's ok, at least this is known.

**Comment by manuelmhtr** [comment]
Hi @mariusa ,
You are right! It was a bug.
It's now solved in [v2.0.2](https://github.com/manuelmhtr/countries-and-timezones/commit/e1f79ee5f56f316b57d3f783b320e7c726467e9d).
